README.md

```diff
@@ -59,12 +59,23 @@ KettleController.power=>UserInterface.powerdisplay
 ```
 Installation
 ============
-TFBrew requires at least Python 3.5 (for asyncio async/await support)
+TFBrew (tfmod2) requires at least Python 3.11 (for asyncio async/await support)
 
 Clone this repository, and set up a virtualenv
 pip install the python packages in the requirements.txt file into your virtualenv
 ```
-pip install -r requirements.txt
+git clone https://github.com/ChuckGl/tfbrew.git
```

Comment:
Pull request into upstream repo should not include directions to clone from a fork.

Reply:
hrafnkelle,

I apologize.  I did not intend a pull request to you.  I am very new to github and python.  I was trying to get ChatGPT to help me convert from pybluez to aioblescan as pybluez cannot be installed on bookworm, the current RPi OS.  ChatGPT give direction to set up a pull on my repo so it could be a contributor.  Guess I did it wrong.  To make matters worse after I did it, ChatGPT said "Oh I cannot I can't be a contributor.  Guess its not quite smart enough yet.  Can you delete the pull or do I need to do something?